domains/Numerical.ml

```diff
@@ -255,36 +330,32 @@ module Numerical(N: NUMERICAL)(C: CONSTRAINT): PARTITION = struct
 
   (**)
 
-  let print fmt b =
```

Comment:
(nitpicking) Can you move 'print' back here? I like to follow the same order used in the module signature (see Partition.ml)